INTITA/Modules/LogInScreen/LogInViewController.swift

```diff
@@ -8,10 +8,11 @@
 import UIKit
 
 enum LoginCells: Int {
```

Comment:
Please use singular for enum names `enum LoginCell: Int { ...`.

Reply:
I have just done it.